packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -43,7 +43,16 @@ export const MenuItem: React.FC<MenuItemProps> = React.memo(
           href={url ? url : undefined}
           target={target}
           className={styles.link}
-          onClick={onClick}
+          onClick={
+            onClick
+              ? (event) => {
+                  if (!(event.ctrlKey || event.metaKey || event.shiftKey) && onClick) {
```

Comment:
@dprokop I wonder if it would make sense to have a `<Link>` component or something for this? I think this is 3rd time I am adding this somewhere.